### PR TITLE
Deprecate Ubuntu-14

### PR DESCRIPTION
**Pull request by christopher-snapp**
Signed-off-by: Christopher A. Snapp <csnapp@chef.io>

### Description

Deprecate Ubuntu-14

### Check List

- [ ] New functionality includes tests
- [ ] All tests pass
- [ ] RELEASE\_NOTES.md has been updated if required (not required for bugfixes, required for API changes)
- [x] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco>

### CODE REVIEW

**Comment by christopher-snapp** [comment]
I've separated the expeditor modifications out to their own commit as requested by @jaymalasinha

File: omnibus/.kitchen.yml [hunk 1]
```diff
@@ -57,6 +57,16 @@ platforms:
     run_list: apt::default
   - name: ubuntu-14.04
     run_list: apt::default
+  - name: ubuntu-14.10
```
Comment (markan) [inline omnibus/.kitchen.yml]: Out of curiosity why are the .10 releases included? Historically we've only built/tested on the LTS releases.

Comment (christopher-snapp) [reply omnibus/.kitchen.yml]: It tripped me up a bit that there weren't *any* 16 or 18 platforms in the kitchen file.  I looked back at the pattern that already existed for 12.x and 13.x and made the assumption that the kitchen file should have all releases beyond just LTS.

Would you like me to go back and remove all of them except 16.04 and 18.04?

**Comment by christopher-snapp** [comment]
Verified ad-hoc build here:

https://buildkite.com/chef/chef-opscode-pushy-client-master-omnibus-adhoc/builds/21

File: omnibus/.kitchen.yml (hunk 1 above)
Comment (markan) [reply omnibus/.kitchen.yml]: Removing everything except 16.04 and 18.04 seems cleanest. Thanks!

Comment (christopher-snapp) [reply omnibus/.kitchen.yml]: Fair enough, I've updated my branch to remove Ubuntu 10 - 14 leaving only the current LTS versions and kicked off the following ad-hoc build for good measure:

https://buildkite.com/chef/chef-opscode-pushy-client-master-omnibus-adhoc/builds/22

**Pull request merged**